app/jobs/api_updater.rb

```diff
@@ -3,6 +3,10 @@ def perform(recurring = true)
     begin
       updater.update
       reschedule if recurring
+      grouper = Grouper.new()
+      grouper.perform(group_genes = should_group_genes?, group_drugs = should_group_drugs?)
```

Comment:
I'm not sure this is doing what you expect - you're declaring a local variable called `group_genes` and setting it to the return value of `should_group_genes?` and then passing that value in as the first positional argument. It would be equivalent to just passing `should_group_genes?` If you were looking to use named arguments, the syntax is slightly different (and I think, unfortunately, incompatible with the version of ruby running on the production server).